package.json

```diff
@@ -33,6 +33,7 @@
   ],
   "dependencies": {
     "@ethersproject/providers": "^5.7.2",
+    "@keyrxng/rpc-handler": "^1.0.1",
```

Comment:
When can we handle deployment to @ubiquitydao @FernandVEYRIER @rndquu

Reply:
I checked and there is no deployment action for https://github.com/ubiquity/rpc-handler so it should be created.
We can also host here indeed but I found it slightly more annoying to use than directly to NPM that worked great through the CLI as well.

Reply:
- https://github.com/ubiquity/rpc-handler/pull/5 

the action exists just waiting to be merged